src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf

```diff
@@ -334,6 +334,58 @@
                 <source>This value should be valid JSON.</source>
                 <target>Dëse Wäert sollt gëlteg JSON.</target>
             </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>Dës Sammlung sollt keng doppelt Elementer enthalen.</target>
```

Comment:
"Double" is not the 100% opposite from "Unique". Could we rewrite this somehow?

Reply:
I checked where it's used and it has pretty much the same meaning, but I'll change it.